tools/configure_travis.sh

```diff
@@ -2,6 +2,16 @@
 
 set -exu
 
+if [ "${TRAVIS_BUILD_TYPE}" = "Release" -a "${CXX}" = "g++-4.9" ]; then
+  # do a pgo build.  for this we do pretty terrible stuff.  make two subdirectories, do
+  # the pgo build + training in one.
+  mkdir ../pyston-build-pgo && cd ../pyston-build-pgo
+  cmake -GNinja -DTEST_THREADS=4 $TRAVIS_BUILD_DIR -DCMAKE_BUILD_TYPE=Release -DENABLE_PGO=ON -DPROFILE_STATE=generate -DPROFILE_DIR=`pwd`/../pyston-build
+  ninja -j4 pyston check-deps
+  ./pyston ${TRAVIS_BUILD_DIR}/minibenchmarks/combined.py
+  cd ../pyston-build
+fi
```

Comment:
I guess I shouldn't have pushed for this; looks like the build times on travis-ci could be prohibitive.  there are things we could do though (reenable ccache for the instrumented build, not enable pgo for llvm so that we can still use ccache for that) but I'm not sure if it's worth it.  as a point of comparison, I don't believe cpython tests the pgo build as part of their regular testing process.